### ISSUE TITLE
Classify_bins confidence value?

### ISSUE THREAD

**Issue by yunha** [bxlab/metaWRAP#52]
I am wondering why my output from classify_bins module does not result in confidence values. 

in the user manual it says "The number next to each classification is the confidance (in percent) of that taxonomy rank classification. As you can see some of the bins are annotated very deeply and others can only be classified as "Bacteria".

I see in my result bin_taxonomy.tab
bin.25.fa	Bacteria
bin.42.fa	Bacteria
bin.3.fa	Bacteria
etc

Is this function not offered anymore?

**Comment by ursky** [comment]
Yeah, it was taken out a few versions ago for simplicity. I'll take that out of the tutorial. The confidence values were unfortunately not terribly reliable.
